Week2/homework/js-exercises/1-remove-the-comma.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+// to get othe length of the string 
+let myNewString = myString.length;
+
+// the output is characters
+console.log(myNewString);
+
+// replace the comma with space
+
+let removeCommaString = myString.replace(/,/g,' ');
+
```

Comment:
`const` here instead of `let` perhaps?

Reply:
thank you so much Yash-kapila I changed to const .